tests/pytest/core/test_session.py

```diff
@@ -8,21 +8,18 @@
 
 
 @pytest.mark.django_db
-def test_active_agency_False(app_request):
+def test_active_agency_False(app_request, model_TransitAgency_inactive):
     session.update(app_request, agency=None)
 
     assert not session.active_agency(app_request)
 
-    agency = models.TransitAgency.objects.first()
-    agency.active = False
-    agency.save()
-
-    session.update(app_request, agency=agency)
+    session.update(app_request, agency=model_TransitAgency_inactive)
 
     assert not session.active_agency(app_request)
 
 
 @pytest.mark.django_db
+@pytest.mark.usefixtures("model_TransitAgency")
 def test_active_agency_True(app_request):
     agency = models.TransitAgency.objects.filter(active=True).first()
```

Comment:
inject the `TransitAgency` needed rather than querying

Reply:
Fixed in [8bdb42d](https://github.com/cal-itp/benefits/pull/788/commits/8bdb42df54bafd3463186009adcc08751664b7c6)